plugins/XmlMaterialProfile/XmlMaterialProfile.py

```diff
@@ -439,6 +439,8 @@ def deserialize(self, serialized):
         meta_data["base_file"] = self.id
         meta_data["status"] = "unknown"  # TODO: Add material verfication
 
+        global_setting_values = {}
```

Comment:
I think the "global" here means the settings in the "settings" section in the XML material file, which can applied to any machine. The settings in "settings/machine" sections are machine specific settings.